app/controllers/target_dashboard_controller.rb

```diff
@@ -6,52 +6,78 @@ def index
     @targets = Aichi11TargetSerializer.new.serialize
     @global_stats = Aichi11Target.get_global_stats
     
-    @row = #[
-      { 
-        title: 'France', 
-        url: 'http://localhost/country/france', 
-        stats: [
-          { 
-            title: 'Coverage',
-            charts: [
-              { 
-                title: 'Terrestrial', value: 55, target: 75, colour: 'terrestrial' 
-              },
-              { 
-                title: 'Marine', value: 55, target: 75, colour: 'marine' 
-              }
-            ]
-          },
-          { 
-            title: 'Effectively managed',
-            charts: [
-              { 
-                title: 'Terrestrial', value: 55, target: 75, colour: 'terrestrial' 
-              },
-              { 
-                title: 'Marine', value: 55, target: 75, colour: 'marine' 
-              }
-            ]
-          },
-          { 
-            title: 'Well connected',
-            charts: [
-              { 
-                title: 'Global', value: 55, target: 75, colour: 'global' 
-              }
-            ]
-          },
-          { 
-            title: 'Areas of importance for biodiversity',
-            charts: [
-              { 
-                title: 'Global', value: 55, target: 75, colour: 'global' 
-              }
-            ]
-          }
-        ] 
-      }.to_json
-    # ]
+
+    ###
+    # need to get text into yml files
```

Comment:
Is this a todo? I usually mark todo as TODO: (because I have an extension that picks that up, but also easy to search by if always the same and codebase will pick this up)